package.json

```
@@ -25,22 +31,30 @@
     "tslib": "^1.10.0"
   },
   "devDependencies": {
+    "remark-cli": "^7.0.1",
+    "remark-preset-lint-recommended": "^3.0.3",
     "prettier": "^1.19.1",
     "typescript": "^3.7.5",
     "rollup": "^1.18.0",
     "rollup-plugin-node-resolve": "^5.2.0",
     "rollup-plugin-sourcemaps": "^0.4.2",
-    "uglify-js": "^3.6.0"
+    "uglify-js": "^3.6.0",
+    "autorest": "^2.0.4413"
+  },
+  "remarkConfig": {
+    "plugins": [
+      "remark-preset-lint-recommended"
+    ]
   },
   "files": [
```

We do not need any globs here, we can just put the directories, also source should not be bundled since it is TS, and README and that stuff is automatic.

```suggestion
  "files": [
    "dist",
    "lib"
  ]
```

---

Again, I think so, but I built it on top of what autorest did... I also found weird being so specific in the files list but I thought "hey, they did it this way for a reason maybe..."